src/util/transactionLocalStorage.ts

```diff
@@ -0,0 +1,39 @@
+// Copyright 2021 Parity Technologies (UK) Ltd.
+// This file is part of Parity Bridges UI.
+//
+// Parity Bridges UI is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Parity Bridges UI is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Parity Bridges UI.  If not, see <http://www.gnu.org/licenses/>.
+
+import { CHAIN_1, CHAIN_2 } from '../configs/substrateProviders';
+import { TransanctionStatus } from '../types/transactionTypes';
+import getSubstrateDynamicNames from './getSubstrateDynamicNames';
+import logger from './logger';
+
+const appendNewTransaction = (transaction: TransanctionStatus) => {
+  const { storageKey } = getSubstrateDynamicNames(`${CHAIN_1}-${CHAIN_2}`);
```

Comment:
I think it would be better off as a separate function. `getSubstrateDynamicNames` has so far always been used by passing one chain name, now we pass some concatentaion and basically all other values of the returned object are never going to be used cause they have no sense at all.